docs/user-guide/third-party-addons/opentracing.md

```diff
@@ -48,6 +49,8 @@ jaeger-sampler-type
 jaeger-sampler-param
 ```
 
+All these options (including host) allow environment variables, such as `$HOSTNAME` or `$HOST_IP`. In the case of Jaeger, if you have a Jaeger agent running on each machine in your cluster, you can use something like `$HOST_IP` (which can be 'mounted' with the `status.hostIP` fieldpath) to make sure traces will be sent to the local agent.
```

Comment:
Please add a link to https://kubernetes.io/docs/tasks/inject-data-application/downward-api-volume-expose-pod-information/#capabilities-of-the-downward-api

Reply:
Done!